src/test/unit/link_quality_unittest.cc

```diff
@@ -426,7 +426,7 @@ TEST(LQTest, TestLQAlarm)
     // elements showing values in alarm range should flash
     simulationTime += 1000000;
     simulationTime -= simulationTime % 1000000;
-    startTime = simulationTime;
+    startTime = simulationTime + 0.25e6;
     for (int i = 0; i < 15; i++) {
```

Comment:
Maybe ?
```suggestion
    startTime = simulationTime;
    for (int i = 1; i <= 15; i++) {
```
or   `simulationTime = startTime + (i + 1) * 0.25e6;`

Reply:
I was just aiming for a minimal change.